src/commands/schema/push.ts

```diff
@@ -34,21 +46,30 @@ export default class PushSchemaCommand extends SchemaCommand {
           // @ts-expect-error-next-line
           duplex: "half",
         });
+
         const json = await res.json();
         if (json.error) {
-          this.error(json.error.message);
+          this.error(json.error?.message ?? json.error);
         }
       } else {
-        // Confirm diff, then push it.
-        const res = await fetch(new URL("/schema/1/validate?force=true", url), {
+        // Confirm diff, then push it. `force` is set on `validate` so we don't
+        // need to pass the last known schema version through.
+        const params = new URLSearchParams({
+          force: "true",
+        });
+        const path = new URL(`/schema/1/validate?${params}`, url);
```

Comment:
should we just drop the "force" param for validate?

Reply:
you can pass a `version` instead, so we just require `force` or `version` to be set on all the schema endpoints. we could probably drop it, it just keeps the validation simple in the core endpoint.

Reply:
right, just thinking it might make using the API itself simpler. I forget/lost track if we're actually passing version to validate at any point

Reply:
I need to double check, I think we might be using it in the dashboard to check if a schema version is stale. If we're not using it there, I'll go ahead and remove it from the core endpoint.